src/payment/PaymentForm.jsx

```diff
@@ -50,6 +37,38 @@ export class PaymentFormComponent extends React.Component {
     // TODO: implement payment submission
   };
 
+  getRequiredFields(fieldValues) {
+    const {
```

Comment:
If you wanted to get super fancy:

```
const {
  state,
  ...requiredFields
} = fieldValues;

if (getStates(requiredFields.country) {
  requiredFields.state = state;
}

return requiredFields;
```

Reply:
I actually need to enumerate the required fields here to filter out the non-required fields.